src/components/Reports/GoalCalculator/GoalCard/useGoalCardCalculations.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import {
+  type GoalLineItemsInput,
+  useGoalLineItems,
+} from '../Shared/useGoalLineItems';
+import { getFamilyTotal } from '../Shared/useReportExpenses/helpers';
+import type { ListGoalCalculationFragment } from '../GoalsList/GoalCalculations.generated';
+
+export const useGoalCardCalculations = (goal: ListGoalCalculationFragment) => {
```

Comment:
Since this only returns the total, what do you think about calling it `useGoalTotal` and returning a number instead of an object?